otherlibs/unix/socketaddr.c

```diff
@@ -103,12 +103,11 @@ void get_sockaddr(value mladr,
 }
 
 value alloc_unix_sockaddr(value path) {
-  value res;
-  Begin_root (path);
-    res = caml_alloc_small(1, 0);
-    Field(res,0) = path;
-  End_roots();
-  return res;
+￼ CAMLparam1(path);
```

Comment:
seems like a [unicode object replacement character](https://www.fileformat.info/info/unicode/char/fffc/index.htm) slipped into the previous line (first codepoint), should instead be a normal whitespace (0x20).